app/components/Views/AccountPermissions/AccountPermissions.tsx

```diff
@@ -110,17 +107,31 @@ const AccountPermissions = (props: AccountPermissionsProps) => {
   const { toastRef } = useContext(ToastContext);
   const [isLoading, setIsLoading] = useState(false);
   const permittedAccountsList = useSelector(selectPermissionControllerState);
-  const permittedAccounts = getPermittedAccountsByHostname(
+  const nonRemappedPermittedAccounts = getPermittedCaipAccountIdsByHostname(
     permittedAccountsList,
     hostname,
   );
-  const permittedChainIds = getPermittedChainIdsByHostname(
+  const permittedAccounts = uniq(
+    nonRemappedPermittedAccounts.map((caipAccountId) => {
+      const {
+        address,
+        chain: { namespace },
+      } = parseCaipAccountId(caipAccountId);
+      if (namespace === KnownCaipNamespace.Eip155) {
+        // this is very hacky, but it works for now
+        return `eip155:0:${address}` as CaipAccountId;
+      }
+      return caipAccountId;
+    }),
+  );
+
+  const permittedChainIds = getPermittedCaipChainIdsByHostname(
```

Comment:
Also perhaps? just to make it easier to follow what is migrated and 
```suggestion
  const permittedCaipChainIds = getPermittedCaipChainIdsByHostname(
```

Reply:
not blocking. I realize these are typed so that's a reasonable alternative reference point